### ISSUE TITLE
Get something wrong when using exchangelib for reading emails with large attachment

### ISSUE THREAD

**Issue by johnvonneumann36** [ecederstrand/exchangelib#670]
Hi guys, i met a strange thing. When i'm trying to load an excel attachment, i cannot get anything right and also cannot not any errors.

```
All_Items = account.inbox.filter(subject__startswith='Test').order_by('-datetime_received')[:1]

for item in All_Items:
    for attachment in item.attachments:
        print(attachment.name)
        if attachment.name.endswith('.xlsx'):
            try:
                df = pd.read_excel(BytesIO(attachment.content))
                print(df)
            except XLRDError: # xlrd.biffh.XLRDError
                print('Error')
            except:
                print('Other error')
        else:
            pass
```
And i got this:
```
Cannot convert value 'UmVjZWl2ZWQ6IGZyb20gQ0gyUFIwN01CNzI1My5uYW1wcmQwNy5wcm9kLm91dGxvb2suY29tICgyNjAzOjEwYjY6NjEwOjVhOjoyMCkNCiBieSBDSDJQUjA3TUI3MjIyLm5hbXByZDA3LnByb2Qub3V0bG9vay5jb20gd2l0aCBIVFRQUyB2aWENCiBDSDJQUjA4Q0EwMDEwLk5BTVBSRDA4LlBST0QuT1VUTE9PSy5DT007IFR1ZSwgMTIgTm92IDIwMTkgMDI6MDQ6MDcgKzAwMDANClJlY2VpdmVkOiBmcm9tIENIMlBSMDdNQjcyMjIubmFtcHJkMDcucHJvZC5vdXRsb29rLmNvbSAoMjAuMTgwLjcuODUpIGJ5DQogQ0gyUFIwN01CNzI1My5uYW1wcmQwNy5wcm9kLm91dGxvb2suY29tICgyMC4xO...
```

The excel file contains 2 columns and 300K rows and can be open by Pandas directly, but cannot be load by exchangelib. If i reduce it to 2 rows, it can be load by exchangelib.

**Comment by ecederstrand** [comment]
Can you please post the full stack trace so we can see where the error message comes from? Just remove the error handling to get the stack trace.

Binary data is Bas64 encoded within EWS, so this should not be Excel-specific.

**Comment by johnvonneumann36** [comment]
![微信截图_20191119090119](https://user-images.githubusercontent.com/48921514/69107089-3d28f080-0aab-11ea-8782-bd392f79693e.png)
Actually i did not get any error, and there is no trace back for this problem. Even the subject of this kind of email can not be decoded. I have attach a screenshot, and hope it is helpful.
And i can open or download or preview excel files with my Outlook client.

**Comment by ecederstrand** [comment]
We are trying to decode the text of the XML element containing the base64-encoded content of the file to normal binary data with `base64.b64decode()`. This fails so we emit the warning. See https://github.com/ecederstrand/exchangelib/blob/04b6ea1f5cd55c0d7fc80f93a6be5edb0ac61aa1/exchangelib/fields.py#L542.

Base64 is a pretty simple protocol, so it's unlikely that there's a bug in `b64decode()`. I assume the problem is that EWS is sending an invalid Base64 string. You can copy-paste the entire string and run it through `b64decode()` to check for yourself. To be absolutely sure, enable debug logging to see the actual XML response and copy-paste the raw string that EWS returns.

**Comment by johnnysprinkles** [comment]
I think I'm hitting this issue as well. I see message like this but can't backscroll far enough in console to see the start of it:

```
...
FeUkyaktqSjRwWEFyaVNDVnZLS2tLVnl3SGY4YWx0NVJISUY0a1ZCMzVJQjdHbFMwRXJZQ0hQUQ0KODRwditvbTNSeG5EZGRvem1uY0NHLzAyS2NDYTNLNFBPM3JnL2hWT0Mxa2lUaGlDQ2VEMHJTRHRFNU9Td0hJWHBrR3JjY2xqZWJrag0KWXJJTTU5TVpxeWVVaWdoTWgzbkJEWTUrbFdCRXBVTWZ1bk9CNzBLSW8yVlFlUWFtVzBLTG1KdHhrUFRQUSsxUXc1U1czV1pXeGpJUA0KVDYvU2ttWGNESTQ4c3IzVUh2M3E5QThzZURJRllqakhkcXZFc2tiQ1NQY3JkaDgzNUNrTkludWJLTFY3WldnbUpNYUR6Q1Y1SUhjVg0KeVFnaktyNVlBNE9BM3AweWE2S3oxS2VBUzI3RWlJNEFHTnBHRHpXZmVSVzZIemxWU3I1UEhQSHBRTXAyanlGUkRPQWg0R1ZPUVJuag0KcjN' on field 'mime_content' to type <class 'str'>
```

exchangelib 2.1 with Python 3.6.9

**Comment by johnnysprinkles** [comment]
Note: I was previously using exchangelib 1.12.1 with Python 2 and never saw any issues like this.

**Comment by ecederstrand** [comment]
@johnnysprinkles Base64-encoded binary data usually ends with one or two equals signs, so it's a bit surprising that your log output doesn't contain one. Your best bet is still to capture the actual content of the XML element and try to decode that. You can always stream your terminal output to a file and handle the data from there.

**Comment by johnvonneumann36** [comment]
I found an discussion about the similar issues, hope it's helpful.

https://social.msdn.microsoft.com/Forums/office/en-US/1d1eb2f5-8adc-4f31-b94b-d83f27788308/exchange-web-services-convert-emailitem-attachment-from-base64-string-to-byte-gives-error?forum=exchangesvrdevelopment

**Comment by ecederstrand** [comment]
Will fix this in #683
